src/apps/dashboard/routes/users/profile.tsx

```diff
@@ -318,6 +327,7 @@ const UserEdit = () => {
                             type='text'
                             id='txtUserName'
                             label='LabelName'
+                            validator={{ pattern: '^([\\w \\-\'._@+]*)([\\w\\-\'._@+])([\\w \\-\'._@+]*)$', errMessage: 'Username must not be empty and contain only numbers, letters, spaces, or the following symbols -\'._@+' }}
```

Comment:
The text of the error message should be moved to `en-us.json` and `globalize.translate` should be used.

Reply:
Is that the way to add new phrases for globalization? Is there anything else I need to do for Weblate?